src/api/animals/AnimalCatFacts.js

```diff
@@ -42,4 +42,4 @@ const Catfacts = () => {
   );
 };
 
-export default Catfacts;
+export default AnimalCatFacts;
```

Comment:
Animals

Reply:
Sorry my Bad I have made the necessary changes